src/modules/user/entities/user.entity.ts

```diff
@@ -1,60 +1,51 @@
 import * as bcrypt from 'bcrypt';
 import { BeforeInsert, Column, Entity, OneToMany } from 'typeorm';
 import { AbstractBaseEntity } from '../../../entities/base.entity';
-import { Testimonial } from '../../../modules/testimonials/entities/testimonials.entity';
+import { Organisation } from '../../../modules/organisations/entities/organisations.entity';
 import { Invite } from '../../invite/entities/invite.entity';
-import { Organisation } from '../../organisations/entities/organisations.entity';
+import { Product } from '../../products/entities/product.entity';
+import { Testimonial } from '../../../modules/testimonials/entities/testimonials.entity';
 
 export enum UserType {
   SUPER_ADMIN = 'super_admin',
   ADMIN = 'admin',
   USER = 'vendor',
 }
-
+@Entity()
```

Comment:
Why do you have this here again even after it has been defined on the next line. remove this